files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2

```diff
@@ -284,4 +284,25 @@ destinations:
             docker_set_user: '1000'
             docker_memory: '{mem}G'
             docker_sudo: false
+  _pulsar_qld_gpu:
+    inherits: _pulsar_destination
+    max_accepted_cores: 24
+    max_accepted_mem: 209
```

Comment:
Where does this number (209) come from?
`free -g` shows that we have a total of 590 G and 584 G available, so 209 would fit 2 of these and still have >100 G remaining.

Reply:
additionally, we have 64 cores available.

Reply:
24/209 was mindless pasting ("note to self get actual numbers")

so we need cores: 64 and mem: 582 (can't be bigger than what is set for slurm node)